packages/connect/src/data/defaultFeeLevels.ts

```diff
@@ -58,20 +60,30 @@ export const getBitcoinFeeLevels = (coin: CoinsJsonData): FeeInfoWithLevels => {
     };
 };
 
-export const getEthereumFeeLevels = (): FeeInfoWithLevels => ({
-    blockTime: -1, // unknown
-    defaultFees: [
-        {
-            label: 'normal' as const,
-            feePerUnit: '5000000000',
-            feeLimit: '21000', // unlike the other networks ethereum have additional value "feeLimit" (Gas limit)
-            blocks: -1, // unknown
-        },
-    ],
-    minFee: 1,
-    maxFee: 10000,
-    dustLimit: -1, // unknown/unused
-});
+export const getEthereumFeeLevels = (chain: string): FeeInfoWithLevels => {
+    const DEFAULT_FEE_PER_UNIT = '5'; // in wei
```

Comment:
this is in `gwei` probably.

Let's set `5` gwei for "gweiChains" and `1` for !"gweiChains"

or if you want, we should also distinguish it by chain.

```
eth: 15,
pol: 200,
bsc: 3, 
base: 0.01
arb: 0.01
op: 0.01
...rest: 1
```